app/models/user.rb

```diff
@@ -25,6 +25,8 @@ class User < ApplicationRecord
 
   accepts_nested_attributes_for :authentications
 
+  scope :active_or_disabled, -> { where(state: %w[active disabled]).order(email: :asc) }
```

Comment:
I'd prefer reverse order, to display most recent users first.

Reply:
fix